api/util.py

```diff
@@ -16,16 +16,7 @@
 
 
 def create_url_from_request(request):
-    # root_path should contain the base path used by a reverse proxy to serve the API. The value of root_path
-    # can be added as parameter to FastAPI() in main.py. By default it is empty.
-    base_path = request.scope.get("root_path")
-
-    # The hostname and scheme (http or https) will (should) be correctly set
-    # from the X-Forwarded-Host and X-Forwarded-Scheme headers by a reverse proxy in front of the API
-    host = request.headers["host"]
-    scheme = request.url.scheme
-
-    return f"{scheme}://{host}{base_path}/collections"
+    return f"{request.base_url}collections"
```

Comment:
```suggestion
    return f"{str(request.base_url)}collections"
```
Making the str conversion explicit.